### ISSUE TITLE
Windows 10 Python 3.[WinError 2] The system cannot find the file specified

### ISSUE THREAD

**Issue by derry43** [glpi-project/tools#49]
**OS:** Windows 10
**python version:** 3.9.11
**glpi-project/tools tag:** 0.4.5
**plugin-release version:** 1.0.5 (not helpful)

_Note: plugin-release script version has not changed since `glpi-project/tools` tag 0.1.7. The plugin-release script version is still 1.0.5, despite numerous modifications to fix issues with python version compatibility_.

All occurrences of `subprocess.Popen` in `tools/plugin-release` require additional argument `shell=True` otherwise we get the following error below on Windows with python 3.9.x.

```
Adding vendor libraries
Traceback (most recent call last):
  File "C:\Users\105061136\Documents\GitHub\ldapcomputers\vendor\glpi-project\tools\tools\plugin-release", line 754, in <module>
    main()
  File "C:\Users\105061136\Documents\GitHub\ldapcomputers\vendor\glpi-project\tools\tools\plugin-release", line 751, in main
    _do_build(repo, buildver)
  File "C:\Users\105061136\Documents\GitHub\ldapcomputers\vendor\glpi-project\tools\tools\plugin-release", line 320, in _do_build
    prepare(plugin_name, archive)
  File "C:\Users\105061136\Documents\GitHub\ldapcomputers\vendor\glpi-project\tools\tools\plugin-release", line 417, in prepare
    p1 = subprocess.Popen(
  File "C:\Users\105061136\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\105061136\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by cedric-anne** [comment]
Hi,

Maybe related to https://github.com/glpi-project/tools/pull/44.

Could you comment the PR, or propose another one ?

Regards

**Comment by derry43** [comment]
Unrelated to #44. Raised a PR.

**Comment by cedric-anne** [comment]
Fixed by 540fdc0391135c8e45d8f1caf48f36c59b89a84b.
